{
  "id": "uno-ooo-parser",
  "version": "0.1.16",
  "libre_office_ver": "7.2",
  "name": "UnsupportedFlavorException",
  "type": "exception",
  "namespace": "com.sun.star.datatransfer",
  "parser_args": {
    "sort": true,
    "long_names": true,
    "remove_parent_inherited": true
  },
  "writer_args": {
    "include_desc": true
  },
  "data": {
    "from_imports": [
      [
        "..uno.exception",
        "Exception",
        "uno_exception_i"
      ]
    ],
    "from_imports_typing": [],
    "extends_map": {
      "com.sun.star.uno.Exception": "uno_exception_i"
    },
    "quote": [],
    "typings": [],
    "requires_typing": false,
    "name": "UnsupportedFlavorException",
    "namespace": "com.sun.star.datatransfer",
    "imports": [],
    "extends": [
      "com.sun.star.uno.Exception"
    ],
    "desc": [
      "Exception will be thrown if there is a request for data in a DataFlavor that is not supported by a data source."
    ],
    "url": "https://api.libreoffice.org/docs/idl/ref/exceptioncom_1_1sun_1_1star_1_1datatransfer_1_1UnsupportedFlavorException.html",
    "items": {}
  }
}